frontend/public/components/factory/Table/_virtualized-table.scss

```diff
@@ -0,0 +1,46 @@
+.pf-c-scrollablegrid .pf-c-table {
```

Comment:
I'm still a little unsure why we need to copy PF CSS into our project. Can you give some background? We probably want a comment here explaining.

Reply:
Hey, sorry @spadgett this is actually a replacement pr for https://github.com/openshift/console/pull/10483 which I inadvertently pushed a wrong commit and then when I tried fixing it couldn't get the failed tests to re run.  

This css is used in their [PF virtual scroll table examples](co-virtualized-table) to function correctly*, but they don't actually ship it with their `react-virtualized-extension`. Here's our earlier comments relating to that https://github.com/openshift/console/pull/10483#pullrequestreview-807782888

When virtualized tables were first added, by Patrick way back when, these rules were added into our `_overrides.scss`.  But at the time we didn't scope them to `<div class="pf-c-scrollablegrid"><table...>`. And it caused James Talton to hit some issues when he was implementing tables. 

It would be better to rename `pf-c-scrollablegrid` to `co-virtualized-table` or something for less confusion, so I'll do that.

* PF css
https://github.com/patternfly/patternfly-react/tree/9866c1a43993d32c3155dfa3851acabe9602c785/packages/react-virtualized-extension/src/components/Virtualized/examples

Reply:
Thanks. I'm still confused, though. Is this required for the virtual table to render correctly at all? If so, shouldn't this be part of the PF extension repo? If not, can you describe how it changes the table?

Reply:
Yes it is confusing, these styles (which were originally in `_overrides.scss`, but not scoped) are required to render our virtualized tables. When I noticed those exact styles where used in the PF virtual scroll table examples https://patternfly-react.surge.sh/extensions/virtual-scroll-table#actions and in these files `@patternfly/react-virtualized-extension/src/components/Virtualized/examples/VirtualGrid.example.css` `@patternfly/react-virtualized-extension/src/components/Virtualized/examples/WindowScroller.example.css` I then reached out to @jcaianirh, and he [posted in PatternFly slack](https://patternfly.slack.com/archives/C4FM977N0/p1637098769062600) asking why those styles were in an `/examples` directory and if we were to recreate them. Their response was it's a community extension and not a core component and doesn't have corresponding styles. 

The styles convert the virtualized `table` to `table-layout: fixed` and each `tr` to `display:table; table-layout: fixed; 
 width: 100%`. And it controls specific padding and widths when used with `.pf-c-table__action` and `.pf-c-table__check`.


https://patternfly.slack.com/archives/C4FM977N0/p1637098769062600

Reply:
Is the layout of PF virtual tables broken for everyone without the styles, or only in our project? In the Slack thread, they mentioned we can contribute the styles back, which seems better if they're really needed.

cc @jcaianirh

Reply:
@sg00dwin I'm OK going ahead with the change to unblock the ACM team, but we should follow up with the PF team to see if we should contribute the styles back upstream in a follow up.

Reply:
@spadgett @sg00dwin Getting some direction from the pf team about contributing the styles back upstream

Reply:
@spadgett @sg00dwin yeah as sam pointed out above the suggestion is since the extension is community run, we're welcome to contribute the styles back to it.